soccer/src/soccer/strategy/agent/position/robot_factory_position.cpp

```diff
@@ -6,7 +6,6 @@ RobotFactoryPosition::RobotFactoryPosition(int r_id) : Position(r_id, "RobotFact
     if (robot_id_ == 0) {
```

Comment:
```suggestion
    if (robot_id_ == goalie_id_) {
```

Reply:
Goalie is never reassigned inside get_task(), so it is important to do this.

Reply:
good catch

Reply:
good catch